huaweicloud/services/acceptance/rds/resource_huaweicloud_rds_database_logs_shrinking_test.go

```diff
@@ -24,15 +24,52 @@ func TestAccRdsDbLogsShrinking_basic(t *testing.T) {
 	})
 }
 
+func testAccRdsDbLogsShrinking_base(name string) string {
+	return fmt.Sprintf(`
+%[1]s
+
+resource "huaweicloud_rds_instance" "test" {
+  name              = "%[2]s"
+  flavor            = "rds.mssql.spec.se.s6.large.2"
+  availability_zone = [data.huaweicloud_availability_zones.test.names[0]]
+  security_group_id = data.huaweicloud_networking_secgroup.test.id
+  subnet_id         = data.huaweicloud_vpc_subnet.test.id
+  vpc_id            = data.huaweicloud_vpc.test.id
+  charging_mode     = "postPaid"
+
+  db {
+    password = "Huangwei!120521"
+    type     = "SQLServer"
+    version  = "2022_SE"
+  }
+
+  volume {
+    type = "ULTRAHIGH"
+    size = 40
+  }
+}
+
+resource "huaweicloud_rds_sqlserver_database" "test" {
+  depends_on  = [huaweicloud_rds_instance.test]
+
+  instance_id = huaweicloud_rds_instance.test.id
+  name        = "%[3]s"
+}
+`, testAccRdsInstance_base(), name, name)
+}
+
 func testAccRdsDbLogsShrinking_basic(name string) string {
 	return fmt.Sprintf(`
 %s
 
 resource "huaweicloud_rds_database_logs_shrinking" "test" {
-  depends_on = [huaweicloud_rds_sqlserver_database.test]
+  depends_on = [
+    huaweicloud_rds_instance.test,
```

Comment:
this resource is not needed here, because the resource huaweicloud_rds_sqlserver_database.test has depends on it